src/general/NewGameSettings.cs

```diff
@@ -703,4 +706,11 @@ private void OnEasterEggsToggled(bool pressed)
     {
         _ = pressed;
     }
+
+    private void OnMetaClicked(object meta)
```

Comment:
I think this should have a slightly more descriptive name like `PerformanceNoteLinkClicked`